src/lib/y2network/udev_rule.rb

```diff
@@ -35,18 +46,23 @@ module Y2Network
   # @example Create a rule from a string
   #   rule = UdevRule.find_for("eth0")
   #   rule.to_s #=> "ACTION==\"add\", SUBSYSTEM==\"net\", ATTR{address}==\"?*31:78:f2\", NAME=\"eth0\""
+  #
+  # @example Writing renaming rules
+  #   rule = UdevRule.new_mac_based_rename("00:12:34:56:78:ab", "eth0")
+  #   UdevRule.write_net_rules([rule])
+  #
+  # @example Writing driver assignment rules
+  #   rule = UdevRule.new_driver_assignment("virtio:d00000001v00001AF4", "virtio_net")
+  #   UdevRule.write_drivers_rules([rule])
+  #
   class UdevRule
     class << self
       # Returns all persistent network rules
       #
       # @return [Array<UdevRule>] Persistent network rules
-      def all
-        return @all if @all
-        rules_map = Yast::SCR.Read(Yast::Path.new(".udev_persistent.net")) || {}
-        @all = rules_map.values.map do |parts|
-          udev_parts = parts.map { |p| UdevRulePart.from_string(p) }
-          new(udev_parts)
-        end
+      def all(group = :net)
```

Comment:
well, I do not like much this all with param. What about helpers like "naming_rules" and "driver_rules"